test/integration/kubernetes.go

```diff
@@ -34,6 +34,9 @@ type k8sEnvironment struct {
 
 // newK8sEnvironment connects to a cluster using kubeconfigPath and creates namespace for the current test.
 func newK8sEnvironment(t *testing.T) k8sEnvironment {
+	defaultBackoff := time.Second
+	defaultTimeout := time.Second * 20
```

Comment:
Are you retrying every second? That's too soon, try out every 30 seconds with a timeout of 120 seconds. Your values are too aggressive. Creating pods sometimes can take a long time

Reply:
So these are the numbers that core-int was using, I just put them into variables since I needed to reuse them. Should I change them numbers for the changes I made only and leave them how core-int had it for their tests?

Reply:
Those two variables are only used in `addPodAndWaitOnPhase`, so go ahead and change the values here. The constants are way too aggressive and will lead to many failed tests

Reply:
Updated to 30 second retries and a 120 second timeout.